From 26ba825324102af0f8151b732fea2a41d2540ce8 Mon Sep 17 00:00:00 2001
From: Sarai Hearthstone <shearthstone@habana.ai>
Date: Tue, 22 Oct 2024 11:05:47 +0000
Subject: [PATCH] Revert "[SW-202574] emb: pyhlml: correct pci domain len"

Revert pyhlml side of support for len 9, which isn't in this branch.

This reverts commit 3641c747dba4a10782dd76103674e92b66be2509.

Change-Id: I01e616f3425fc08639ae1695c7511ddb22d48e7a
---

diff --git a/pyhlml/pyhlml/hlml_types.py b/pyhlml/pyhlml/hlml_types.py
index 8ccb0b5..5d7c2a4 100644
--- a/pyhlml/pyhlml/hlml_types.py
+++ b/pyhlml/pyhlml/hlml_types.py
@@ -8,7 +8,7 @@
     TYPE                                    = ctypes.c_void_p
 
 class HLML_DEFINE:
-    PCI_DOMAIN_LEN                          = 9
+    PCI_DOMAIN_LEN                          = 5
     PCI_ADDR_LEN                            = ( PCI_DOMAIN_LEN + 10 )
     PCI_LINK_INFO_LEN                       = 10
     HL_FIELD_MAX_SIZE                       = 32
